scripts/windows/setup/Install-SecurityDaemon.ps1

```diff
@@ -1,32 +1,41 @@
+new-module -name IoTEdge -scriptblock {
+
+[Console]::OutputEncoding = New-Object -typename System.Text.ASCIIEncoding
+
 <#
  # Installs the IoT Edge Security Daemon on RS4 Windows.
  #>
 
 #requires -Version 5
 #requires -RunAsAdministrator
 
-[CmdletBinding()]
-param (
-    [Parameter(ParameterSetName = "Manual")]
-    [Switch] $Manual,
-
-    [Parameter(ParameterSetName = "DPS")]
-    [Switch] $Dps,
-
-    [Parameter(Mandatory = $true, ParameterSetName = "Manual")]
-    [String] $DeviceConnectionString,
-
-    [Parameter(Mandatory = $true, ParameterSetName = "DPS")]
-    [String] $ScopeId,
 
-    [Parameter(Mandatory = $true, ParameterSetName = "DPS")]
-    [String] $RegistrationId
-)
 
 $ErrorActionPreference = "Stop"
 Set-StrictMode -Version 5
 
 function Install-SecurityDaemon {
+    [CmdletBinding()]
```

Comment:
With this change you can no longer pass arguments to the script on the command line; you have to dot-source the script and then call the function with args. Is that intended?

Reply:
Yes, mainly for convenience for the automated install. If not, user has to be first download the file and then tinker with set-executionpolicy to get it run.